### PR TITLE
Dev rares

### PR DESCRIPTION
**Pull request by OpariucRares**
added UI for MedicalRecord and made the MedicalRecord 100%

### CODE REVIEW

File: src/DucksNet.WebUI/Pages/MedicalRecordOverview.razor [hunk 1]
```diff
@@ -1,29 +1,169 @@
...
+                    private void HandleValidSubmit()
+                    {
+                        CreateMedicalRecord(createMedicalRecord);
+                    }
+                }
+            </div>
+        </div>
+    </div>
+    <div class="card">
+        <div class="card-header" id="headingThree">
```
Comment (Al-Andrew) [inline src/DucksNet.WebUI/Pages/MedicalRecordOverview.razor]: I think the ID here should be headingFour. but the browser renders so... todo

File: src/DucksNet.WebUI/Pages/Models/UpdateMedicalRecord.cs [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+namespace DucksNet.WebUI.Pages.Models;
+
+public class UpdateMedicalRecord
+{
+    [Required]
+    [StringLength(36, MinimumLength = 36, ErrorMessage = "Guid must be 36 characters long.")]
+    public string? MedicalRecordId { get; set; }
+
+    [StringLength(36, MinimumLength = 36, ErrorMessage = "Id Appointment must be 36 characters long.")]
+    public string? IdAppintment { get; set; }
```
Comment (Al-Andrew) [inline src/DucksNet.WebUI/Pages/Models/UpdateMedicalRecord.cs:12]: What happens if you pass null to the backend here?

**Review by Al-Andrew — APPROVED** [review]
good enough

File: src/DucksNet.WebUI/Pages/Models/UpdateMedicalRecord.cs (hunk 2 above)
Comment (OpariucRares) [reply src/DucksNet.WebUI/Pages/Models/UpdateMedicalRecord.cs:12]: it checks in the update method if the guid == guid.empty

**Pull request merged**